code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -595,43 +595,51 @@
-		/obj/item/reagent_containers/hypospray/autoinjector/quickclot = 4,
-		/obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus = 0,
-		/obj/item/reagent_containers/hypospray/autoinjector/quickclotplus = 0,
-		/obj/item/reagent_containers/hypospray/autoinjector/synaptizine = 0,
-		/obj/item/storage/pill_bottle/bicaridine = 3,
-		/obj/item/storage/pill_bottle/kelotane = 3,
-		/obj/item/storage/pill_bottle/tramadol = 3,
-		/obj/item/storage/pill_bottle/tricordrazine = 3,
-		/obj/item/storage/pill_bottle/inaprovaline = 3,
-		/obj/item/storage/pill_bottle/dexalin = 3,
-		/obj/item/storage/pill_bottle/dylovene = 3,
-		/obj/item/storage/pill_bottle/spaceacillin = 3,
-		/obj/item/storage/pill_bottle/alkysine = 3,
-		/obj/item/storage/pill_bottle/imidazoline = 3,
-		/obj/item/storage/pill_bottle/peridaxon = 2,
-		/obj/item/storage/pill_bottle/quickclot = 2,
-		/obj/item/storage/pill_bottle/hypervene = 2,
-		/obj/item/stack/medical/advanced/bruise_pack = 8,
-		/obj/item/stack/medical/bruise_pack = 8,
-		/obj/item/stack/medical/advanced/ointment = 8,
-		/obj/item/stack/medical/ointment = 8,
-		/obj/item/stack/medical/splint = 4,
-		/obj/item/healthanalyzer = 3,
-		/obj/item/bodybag/cryobag = 2,
+		"Pill Bottle" = list(
+			/obj/item/storage/pill_bottle/bicaridine = 12,
+			/obj/item/storage/pill_bottle/kelotane = 12,
+			/obj/item/storage/pill_bottle/tramadol = 12,
+			/obj/item/storage/pill_bottle/tricordrazine = 12,
+			/obj/item/storage/pill_bottle/inaprovaline = 12,
+			/obj/item/storage/pill_bottle/dexalin = 12,
+			/obj/item/storage/pill_bottle/dylovene = 12,
+			/obj/item/storage/pill_bottle/spaceacillin = 12,
+			/obj/item/storage/pill_bottle/alkysine = 12,
+			/obj/item/storage/pill_bottle/imidazoline = 12,
+			/obj/item/storage/pill_bottle/peridaxon = 8,
+			/obj/item/storage/pill_bottle/quickclot = 8,
+			/obj/item/storage/pill_bottle/hypervene = 8,
+		),
+		"Auto Injector" = list(
+			/obj/item/reagent_containers/hypospray/autoinjector/bicaridine = 24,
+			/obj/item/reagent_containers/hypospray/autoinjector/kelotane = 24,
+			/obj/item/reagent_containers/hypospray/autoinjector/tramadol = 24,
+			/obj/item/reagent_containers/hypospray/autoinjector/oxycodone = 12,
+			/obj/item/reagent_containers/hypospray/autoinjector/tricordrazine = 32,
+			/obj/item/reagent_containers/hypospray/autoinjector/combat = 8,
+			/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 24,
+			/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 24,
+			/obj/item/reagent_containers/hypospray/autoinjector/hypervene = 16,
+			/obj/item/reagent_containers/hypospray/autoinjector/isotonic = 16,
+			/obj/item/reagent_containers/hypospray/autoinjector/quickclot = 16,
+			/obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus = 0,
+			/obj/item/reagent_containers/hypospray/autoinjector/quickclotplus = 0,
+			/obj/item/reagent_containers/hypospray/autoinjector/synaptizine = 0,
+		),
+		"Heal Pack" = list(
```

Comment:
dunno how to call those actually, heal pack sound weh
```suggestion
		"First Aid" = list(
```

Reply:
heal packs is what it is though